src/test/java/com/redhat/lightblue/camel/utils/LightblueInsertRequestTransformer.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.lightblue.camel.transformer;
+package com.redhat.lightblue.camel.utils;
```

Comment:
@paterczm  this file should be moved to src/main too, and should be changed to AbstractLightblueDataRequestTransformer that supports insert/update/save operation as an acceptable input param